### PR TITLE
some helper functions

### PR DESCRIPTION
**Pull request by sideeffffect**

### CODE REVIEW

File: src/FSharpx.Collections/Collections.fs [hunk 1]
```diff
@@ -234,26 +234,21 @@ module Seq =
 
     /// Extracts from a seq of Choice all the Choice1Of2 elements. All the Choice1Of2 elements are extracted in order.
     let inline choice1s xs =
-        let chooser = function
-                      | Choice1Of2 x -> Some x
-                      | _            -> None
+        let inline chooser a = match a with
```
Comment (panesofglass) [inline src/FSharpx.Collections/Collections.fs]: Why the use of `inline` here? Did this help with performance or compilation in some way? I'm not arguing against it; I'm just curious.

Comment (sideeffffect) [inline src/FSharpx.Collections/Collections.fs]: yes, it seemed that having true (un-inlined) functions here is not necessary and that inlining them could help performance. but I don't have any number to back it up.

**Comment by panesofglass** [comment]
ping ... seems like this is a necessary fix to get other PR builds cleared.

**Comment by jackfoxy** [comment]
changing name of PR as it no longer has to do with fixing Travis build

File: src/FSharpx.Collections/Collections.fs (hunk 1 above)
Comment (jackfoxy) [reply src/FSharpx.Collections/Collections.fs]: Was not the whole choide1s already inlined? 
Same question for similar changes below.

Comment (sideeffffect) [reply src/FSharpx.Collections/Collections.fs]: but not the helper function `chooser`

**Comment by jackfoxy** [comment]
@sideeffffect  I quite possibly botched the conflict resolution. If so. Please submit a new PR.

**Pull request closed without merging**